packages/grafana-schema/src/schema/graph.gen.ts

```diff
@@ -11,185 +10,231 @@ export enum AxisPlacement {
   Right = 'right',
   Top = 'top',
 }
+
 export enum VisibilityMode {
-  Auto = 'auto',
   Always = 'always',
+  Auto = 'auto',
   Never = 'never',
 }
-export enum GraphDrawStyle {
+
+export enum DrawStyle {
```

Comment:
this seems to have the wrong name (should be GraphDrawStyle), since we have no way currently to namespace names in the ui npm packages we have to prefix names to make them a bit more specific, see that we are very inconsistent with that here (since before cue) .

Reply:
I think I fixed this in https://github.com/grafana/grafana/pull/39694/commits/63b33bdc462e702902fd1c8900d514a613381077, but will let sam regenerate